src/foundry/foundry.js/pixi/containers/placeableObjects/measuredTemplate.d.ts

```diff
@@ -59,9 +47,11 @@ declare global {
     /**
      * A convenience accessor for the fill color as a numeric hex code
      */
-    get fillColor(): string;
+    get fillColor(): number;
```

Comment:
I think this (and `borderColor` can still return (numeric) strings, right? If `this.data.borderColor` is set, it will return a string of the form `0x...`.

Reply:
Changed it to `string | number`

Reply:
Changed it to `string | number`